job-service-client2/pom.xml

```diff
@@ -32,14 +32,10 @@
         <relativePath/>
     </parent>
 
-    <organization>
-        <name>EntIT Software LLC, a Micro Focus company</name>
-        <url>https://www.microfocus.com</url>
-    </organization>
+    <inceptionYear>2016</inceptionYear>
```

Comment:
Why is this changing?

Reply:
Its not needed, is inherited from the parent.